qemu/tests/cfg/blockdev_stream_backing_mask_on.cfg

```diff
@@ -0,0 +1,65 @@
+# The following testing scenario is covered:
+#   Basic block stream test based on an existed snapshot
+#     data -> sn1 -> sn2 -> sn3 -> sn4
+#     The snapshot images are local fs images
+#     block-strem: {device: sn4, base-node: data}
+
+- blockdev_stream_backing_mask_on:
+    only Linux
+    qemu_force_use_drive_expression = no
+    type = blockdev_stream_backing_mask_on
+    virt_test_type = qemu
+    start_vm = yes
+    kill_vm = yes
+    images += " data"
+    base_tag = data
+    node = drive_data
+    remove_image_data = yes
+    force_create_image_data = yes
+    snapshot_images = "sn1 sn2 sn3 sn4"
+    snapshot_tag = sn4
+    base_node = drive_data
+    tempfile_size = 200M
+    storage_pools = default
+    storage_pool = default
+    rebase_mode = unsafe
+    required_qemu = [9.0.0,)
+
+    image_size_data = 2G
+    image_size_sn1 = ${image_size_data}
+    image_size_sn2 = ${image_size_data}
+    image_size_sn3 = ${image_size_data}
+    image_size_sn4 = ${image_size_data}
+    image_name_data = data
+    image_name_sn1 = sn1
+    image_name_sn2 = sn2
+    image_name_sn3 = sn3
+    image_name_sn4 = sn4
+    image_format_data = raw
+    image_format_sn1 = qcow2
+    image_format_sn2 = qcow2
+    image_format_sn3 = qcow2
+    image_format_sn4 = qcow2
+    backing_mask_protocol = yes
+    gluster_direct:
+        enable_gluster_data = yes
```

Comment:
can we remove this because we don't support it in 9.0, but we need to keep it if upstream still supports it

Reply:
please also check https://github.com/autotest/tp-qemu/pull/4141

Reply:
hi @aliang123 , what about this one?